app/types/crashes.ts

```diff
@@ -69,4 +70,5 @@ export type Crash = {
   units: Unit[] | null;
   charges_cris: Charge[] | null;
   people_list_view: Person[] | null;
+  crashes_list_view: CrashesListRow;
```

Comment:
idk if this is the right way to handle this, i didnt know if i should use CrashesListRow or create a new type for this? lmk @johnclary

Reply:
yep, this looks great to me. the more our types resemble our GraphQL objects the better ✨